setup.py

```diff
@@ -1,15 +1,44 @@
-from os import path
+import os
+from distutils.errors import CompileError
 from setuptools import setup
+from subprocess import call
+from distutils.command.build import build
 
-with open(path.join(path.abspath(path.dirname(__file__)), 'statsig', 'version.py')) as f:
+with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'statsig', 'version.py')) as f:
     exec(f.read())  # pylint: disable=exec-used
 
-with open(path.join(path.abspath(path.dirname(__file__)), 'README.md')) as r:
+with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'README.md')) as r:
     README = r.read()
 
+class StatsigBuild(build):
+    def run(self):
+        clean = [
+            'make',
+            '-C',
+            'statsig/shared/',
+            'clean',
+        ]
+
+        make = [
+            'make',
+            '-C',
+            'statsig/shared/',
+            'install',
+        ]
```

Comment:
now, instead of bundling the .so file with the library, we build it when pip install is run.